assets/section-featured-blog.css

```diff
@@ -7,21 +7,50 @@
 }
 
 .blog .placeholder {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 22rem;
-  text-align: center;
-  padding: 4rem 2rem 5rem;
-  margin: 0 2rem;
+  margin: 0 1.5rem;
 }
 
 @media screen and (min-width: 750px) {
+  .blog .placeholder {
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 990px) {
   .blog .placeholder {
     margin: 0;
   }
 }
 
+.blog .placeholder .placeholder-content {
+  padding: 3rem;
+}
+
+.blog .placeholder .placeholder-image {
+  background: rgba(var(--color-foreground), .04);
+  overflow: hidden;
+  position: relative;
+  padding-bottom: 66.6%;
+}
```

Comment:
This seems quite similar to the `.media` class. Could use it instead maybe: `media media--transparent media--landscape`